matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/session/room/send/LocalEchoEventFactory.kt

```diff
@@ -612,7 +612,7 @@ internal class LocalEchoEventFactory @Inject constructor(
         val userId = eventReplied.root.senderId ?: return null
         val userLink = permalinkFactory.createPermalink(userId, false) ?: return null
 
-        val body = bodyForReply(eventReplied.getLastMessageContent(), eventReplied.isReply(), isRedactedEvent)
+        val body = bodyForReply(eventReplied, isRedactedEvent)
```

Comment:
I see there is the same logic in `createReplaceTextOfReply` method for the case of editing a reply message. Maybe we should do the same modification as here, no?

Reply:
Make sense, but need testing.

Reply:
Fixed in [7a95ece](https://github.com/vector-im/element-android/pull/7969/commits/7a95ece7381c17802e718f6473ca574fea5d621a)